app/views/dit_landing_page/_training.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="landing-page__section dit-landing-page__section--dark-grey landing-page__section--border-bottom-red">
+  <div class="govuk-width-container">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-one-half">
+        <h2 class="govuk-heading-l">
+          <%= t("dit_landing_page.training_section_title") %>
+        </h2>
+        <%= render "govuk_publishing_components/components/govspeak", {
+        } do %>
+          <%= t("dit_landing_page.training_section_description").html_safe() %>
+        <% end %>
+      </div>
+      <div class="govuk-grid-column-one-half">
+        <%= image_tag 'DIT-small-image.png' %>
```

Comment:
I understand this is not the final image but, as above it requires an `alt` attribute – if the image is purely decorative you can provide a blank string so that the screen readers will ignore it (otherwise it will announce that there is an image and possibly give its dimensions, leaving the user wondering what the image is and if it is important). To make it responsive I added a class to ensure it doesn't overflow the container. Given it's further down the page we can also make use of the lazy-loading feature available in modern browsers that might save mobile users a request.
```suggestion
        <%= image_tag 'DIT-small-image.png', class: "dit-landing-page__training-image", alt: "", loading: "lazy" %>
```
with
```
.dit-landing-page__training-image {
  max-width: 100%;
}
```

Reply:
I'm not familiar with the `loading` option for the `image_tag` helper and I can't find any documentation. There's [a gem](https://github.com/jassa/lazyload-rails), but it uses slightly different syntax, ie `lazy:true`. Is there a gem you had in mind?

Reply:
No additional code required, it's [an html attribute](https://developer.mozilla.org/en-US/docs/Web/Performance/Lazy_loading#Images_and_iframes) applied to the `<img>` element. When browsers support this users will benefit from the feature, when not no harm is done.

Reply:
OK I see! Thanks for explaing!